test/unit/lodash.spec.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function ()
+    {
+        describe('clamp', function ()
+        {
+
+            var params, number, lower, upper;
+
+            before(function ()
+            {
+                params = datasets.clamp();
+                number = params[0];
+                lower = params[1];
+                upper = params[2];
+            });
+
+            it('should verify params', function ()
+            {
+                expect(params).to.have.length(3);
+                expect(lower).to.be.below(upper);
+                expect(typeof number).to.eql('number');
+                expect(typeof lower).to.eql('number');
+                expect(typeof upper).to.eql('number');
+            });
+
+            it('should return the clamped number', function ()
+            {
+                expect(_.clamp.apply(_, params)).to.eql(3);
+            });
+        });
```

Comment:
In fact we can have two scenarios with this method:
1. when first param is lower then second param
2. when first param is higher then last param.
We should check both. I thing It will be good way to have two methods in datasets: clamp1 and clamp2. Then we can place two new describes inside this main 'clamp' describe. Texts of them can be same like those above. And accordingly in first describe user will have to provide params where first is lower then second. Similar situation in second describe.